cypress/e2e/0-JoinBet/registration/registration.cy.js

```diff
@@ -37,4 +38,80 @@ describe('Registration. Localization. Casino. Header.',() => {
                 .click();
         });
     });
+});
+
+describe('Registration. Casino. Header.', () => {
+    beforeEach(() => {
+        cy.authenticateUrl('https://st2-gdjsmns.hk/ru/casino/lobby');
+        bonusModalClose();
+    });
+    /*
+    C4928 - https://jbtest.testrail.io/index.php?/cases/view/4928
+     */
+    it('Проверить открытие/закрытие дропдауна "Валюта"', () => {
+        cy.get(header.regButton)
+            .click();
+        cy.get(header.window1.currencyItem)
+            .click();
+        cy.get(header.window1.currencyList)
+            .should('be.visible')
+            .get(header.window1.banner)
+            .click()
+            .get(header.window1.currencyList)
+            .should('not.be.visible');
+        cy.get(header.closeRegButton)
+            .click();
+    });
+    /*
+    C1501 - https://jbtest.testrail.io/index.php?/cases/view/1501
+     */
+    it('Нажать на кнопку "Регистрация по телефону" после заполнения полей',() => {
+        cy.get(header.regButton)
+            .click();
+        cy.get(header.window1.email)
+            .type(100)
+            .should('have.value', 100)
+            .and('be.visible');
+        cy.get(header.window1.countryButton)
+            .click()
+            .get(header.window1.country)
+            .eq(1)
+            .click()
+            .get(header.window1.countryButton)
+            .should('have.text', 'Австрия');
```

Comment:
@ElKalinin  1 more example:
```suggestion
        cy.get(header.window1.countryButton).click()
        cy.get(header.window1.country).eq(1).click()
        cy.get(header.window1.countryButton).should('have.text', 'Австрия');
```

Reply:
done